test/svymean.jl

```diff
@@ -1,19 +1,30 @@
 @testset "svymean.jl" begin
     # SimpleRandomSample
+    using DataFrames
```

Comment:
This line should not be here. If you need to use `DataFrames` functionality use `DataFrames.` as prefix.